packages/cms/src/plugins/sanity-deploy-button/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+import Button from 'part:@sanity/components/buttons/default';
+import { useSecrets, SettingsView } from 'sanity-secrets';
+
+import styles from './deploy.css';
+
+const namespace = 'github-actions-deployment';
+
+const pluginConfigKeys = [
+  {
+    key: 'webhookUrl',
+    description: 'The webhook to trigger a deployment',
+    title: 'The webhook to trigger a deployment',
+  },
+];
+
+type Secrets =
+  | {
+      webhookUrl?: string;
+    }
+  | undefined;
+
+function Deploy() {
+  const { secrets }: { secrets: Secrets } = useSecrets(namespace);
```

Comment:
this might read easier:
```suggestion
  const secrets = useSecrets(namespace).secrets as Secrets;
```